jest.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
The jest config is provided in `package.json` so we need to remove one of them.

Reply:
So I could debug in vs-code I chose to move all jest config from package.json to jest.config.js. Is this ok? Thanks for pointing it out

Reply:
Yeah, that's fine. 👍 
Since you already use `ts-jest`, I think you can remove some of the configs already. Like `transform` which is included in `preset: ts-jest` by default.